src/GHAppy.hs

```diff
@@ -301,7 +306,7 @@ compose = reinterpret go
       AddAllPagesThat lvl p ->
         let f = fmap ((\n -> emptyLeaf {issueN = n, level = lvl}) . Just . fst) . M.toList . M.filter (getPredicate p) . unIssues
          in (get >>= tell . f)
-      AddHeader lvl str -> tell [emptyLeaf {preamble = [replicate (fromEnum lvl) '#' <> " " <> str]}]
+      AddHeader lvl str -> tell [emptyLeaf {preamble = [T.pack $ replicate (fromEnum lvl) '#' <> " " <> T.unpack str]}]
```

Comment:
```suggestion
      AddHeader lvl str -> tell [emptyLeaf {preamble = [T.pack (replicate (fromEnum lvl) '#') <> " " <> str]}]
```
If you move the bracket you avoid packing and unpacking - also it helps to have that `OverloadedStrings` on